docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+This how-to describes configuration of the Nodeos `backing store`. Nodeos can now use chainbase or rocks-db as a backing store for smart contract state.
+   
+# Prerequisites
+Version 2.1 or above of the EOSIO development environment. 
+
+# Parameter Definitions 
+Specify which backing store to use with the chain_plugin --backing-store argument. This argument sets state storage to either `chainbase`, the default, or `rocksdb`.
```

Comment:
chainbase, rocksdb - same comment as above we should strive to be consistent how we capitalize it.

Reply:
Case in this case is based on parameter use.